app/pages/multisig/updates/PoolParameters/UpdatePoolParameters.tsx

```diff
@@ -0,0 +1,256 @@
+                    defaultValue={
+                        defaults.finalizationCommissionLPool ||
+                        finalizationCommissionLPool.toString()
+                    }
+                    rules={validationRules(
+                        fieldDisplays.finalizationCommissionLPool
+                    )}
+                />
+                <FormCommissionRangeField
+                    name={fieldNames.transactionCommissionRange}
+                    label={fieldDisplays.transactionCommissionRange}
+                    defaultValue={
+                        defaults.transactionCommissionRange ||
+                        transactionCommissionRange
+                    }
+                    rules={rangeValidationRules(
+                        fieldDisplays.transactionCommissionRange
+                    )}
+                />
+                <FormCommissionRangeField
+                    name={fieldNames.bakingCommissionRange}
+                    label={fieldDisplays.bakingCommissionRange}
+                    defaultValue={
+                        defaults.bakingCommissionRange || bakingCommissionRange
+                    }
+                    rules={rangeValidationRules(
+                        fieldDisplays.bakingCommissionRange
+                    )}
+                />
+                <FormCommissionRangeField
+                    name={fieldNames.finalizationCommissionRange}
+                    label={fieldDisplays.finalizationCommissionRange}
+                    defaultValue={
+                        defaults.transactionCommissionRange ||
+                        finalizationCommissionRange
+                    }
+                    rules={rangeValidationRules(
+                        fieldDisplays.finalizationCommissionRange
+                    )}
+                />
+                <FractionFieldForm
+                    label={fieldDisplays.capitalBound}
+                    name={fieldNames.capitalBound}
+                    defaultValue={
+                        defaults.capitalBound || capitalBound.toString()
+                    }
+                />
+                <ErrorMessage>{firstError}</ErrorMessage>
+                <FormRelativeRateField
+                    name={fieldNames.leverageBound}
+                    label={fieldDisplays.leverageBound}
+                    defaultValue={
+                        defaults.leverageBound || {
+                            numerator: leverageBound.numerator.toString(),
+                            denominator: leverageBound.denominator.toString(),
+                        }
+                    }
+                    numeratorUnit={{
+                        value: ' max delegator stake',
+                        position: 'postfix',
+                    }}
+                    denominatorUnit={{
+                        value: ' equity capital',
+                        position: 'postfix',
+                    }}
```

Comment:
Looks a bit weird to me that the labels describing what each number represents are included as the units of each side. It would make more sense to me, if it was included in in the label of the field instead.

Ideally, another representation of the relation would be used. I don't think the `X=Y` representation makes as much sense here as when used to describe a relation between amounts of different units. Here, we should use something that communicates `X/Y` instead.

Maybe to update the component to display as either one or the other?